tests/verkle/eip4762_verkle_gas_witness/test_balance.py

```diff
@@ -57,7 +57,7 @@ def test_balance(blockchain_test: BlockchainTestFiller, fork: Fork, target, warm
     ],
```

Comment:
From this point forward, the changes are mostly the new tests so nothing framework related.